noir-projects/aztec-nr/aztec/src/context/private_context.nr

```diff
@@ -16,7 +16,7 @@ use dep::protocol_types::{
     private_circuit_public_inputs::PrivateCircuitPublicInputs,
     public_call_stack_item::PublicCallStackItem,
     public_circuit_public_inputs::PublicCircuitPublicInputs, read_request::ReadRequest,
-    side_effect::{SideEffect, SideEffectLinkedToNoteHash}
+    note_hash::NoteHash, nullifier::Nullifier, side_effect::SideEffect
```

Comment:
Creating specific structs for note hashes and nullifiers. Logs are still using SideEffect because Miranda is working on logs so I don't want to change it to create more conflicts.